### PR TITLE
31 displaying distance

### LINKED ISSUE #31: Displaying the distance
A ingame meter that shows the distance ran in units

### PR DESCRIPTION
**Pull request by mdingjan**
Will close #31 

Ready for review.

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/level/Track.java [hunk 1]
```diff
@@ -111,6 +117,53 @@ public final Player getPlayer() {
...
+	void setDistance(final int dist) {
+		Track.distance = dist;
+	}
+		
+	/**
+	 * Updates the current distance every 50 moves.
+	 * @return updated distance
+	 */
+	public int moduloDistance() {
+		if ((distance % 50) == 0) {
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/level/Track.java]: Magic number; better extract "50" to a constant (then you can also use this constant in the test suite and other places if needed).

Also, why not just `return ((int) distance) % 50;` ? Then you don't have to keep updating the `modDistance` field :)

**Comment by mdingjan** [comment]
@mpsijm Thank you, I processed your feedback :D

File: src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java [hunk 2]
```diff
@@ -32,6 +32,10 @@ private void renderScene() {
 		GameWindow.addWorld(entities);
 
 		GameWindow.addOverlay(new Text(0, 16, "Score: " + State.getScore()));
+		GameWindow.addOverlay(new Text(0, 32, "Distance: " 
+				+ State.getTrack().moduloDistance()));
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java]: Maybe you could also move moduloDistance() to State? It looks a bit weird that public getDistance() is moved to State but this one isn't :P

Comment (mdingjan) [inline src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java]: That's true ;) I will change it now.

**Pull request merged**